ihmc-high-level-behaviors/src/test/java/us/ihmc/rdx/perception/RDXRapidHeightMapExtractorCudaDemo.java

```diff
@@ -0,0 +1,199 @@
+package us.ihmc.rdx.perception;
+
+import org.bytedeco.javacpp.BytePointer;
+import org.bytedeco.opencv.global.opencv_core;
+import org.bytedeco.opencv.opencv_core.GpuMat;
+import org.bytedeco.opencv.opencv_core.Mat;
+import perception_msgs.msg.dds.HeightMapMessage;
+import perception_msgs.msg.dds.ImageMessage;
+import us.ihmc.commons.thread.Throttler;
+import us.ihmc.communication.PerceptionAPI;
+import us.ihmc.euclid.referenceFrame.FramePose3D;
+import us.ihmc.euclid.referenceFrame.ReferenceFrame;
+import us.ihmc.euclid.referenceFrame.tools.ReferenceFrameTools;
+import us.ihmc.euclid.transform.RigidBodyTransform;
+import us.ihmc.log.LogTools;
+import us.ihmc.perception.camera.CameraIntrinsics;
+import us.ihmc.perception.gpuHeightMap.RapidHeightMapExtractor;
+import us.ihmc.perception.gpuHeightMap.RapidHeightMapExtractorCuda;
+import us.ihmc.perception.opencv.OpenCVTools;
+import us.ihmc.perception.tools.NativeMemoryTools;
+import us.ihmc.perception.tools.PerceptionMessageTools;
+import us.ihmc.rdx.Lwjgl3ApplicationAdapter;
+import us.ihmc.rdx.mesh.RDXMultiColorMeshBuilder;
+import us.ihmc.rdx.sceneManager.RDXSceneLevel;
+import us.ihmc.rdx.simulation.environment.RDXEnvironmentBuilder;
+import us.ihmc.rdx.simulation.sensors.RDXSensorSimulator;
+import us.ihmc.rdx.ui.RDXBaseUI;
+import us.ihmc.rdx.ui.gizmo.RDXPose3DGizmo;
+import us.ihmc.rdx.ui.graphics.RDXHeightMapGraphicNew;
+import us.ihmc.robotics.referenceFrames.ZUpFrame;
+import us.ihmc.sensorProcessing.heightMap.HeightMapData;
+import us.ihmc.sensorProcessing.heightMap.HeightMapMessageTools;
+
+import java.util.List;
+
+public class RDXRapidHeightMapExtractorCudaDemo
```

Comment:
```suggestion
public class RDXRapidHeightMapExtractorCUDADemo
```
Or whatever change to match the name change of the extractor